lua/modules/uikit.lua

```diff
@@ -660,6 +663,39 @@ function createUI(system)
 					end
 				end
 			end
+		elseif k == "onClick" then
+			if t.type == NodeType.Button then
+				t._onClick = function(self)
+					_buttonOnRelease(self, v)
+				end
+			elseif t.type == NodeType.Frame then
+				local background = t.background
+				if v == nil then
+					t._onClick = nil
+					if t._onPress == nil then
```

Comment:
what if _onRelease is set?
We should not disable collisions

Reply:
_onClick should also be tested when setting _onRelease and _onPress to nil

Reply:
I just updated this, thanks for finding that one!